services/app-web/src/pages/StateSubmission/EQROSubmission/EQROSubmissionDetails/EQROSubmissionDetails.tsx

```diff
@@ -22,29 +56,27 @@ export const EQROSubmissionDetails = (): React.ReactElement => {
 
     const { createDraft } = useContractForm(id)
 
+    const [shouldValidate, setShouldValidate] = useState(false)
+    const showFieldErrors = (error?: FormError) =>
+        shouldValidate && Boolean(error)
+
     const activeMainContentId = 'submissionDetailsPageMainContent'
 
     // Set the active main content to focus when click the Skip to main content button.
     useEffect(() => {
         updateActiveMainContent(activeMainContentId)
     }, [activeMainContentId, updateActiveMainContent])
 
-    /**
-     * Most of this code here is placeholder code so we can progress to the next form page.
-     * It will create a new draft EQRO submission, but no updates or validations.
-     */
-    const onSubmit = async (e: FormEvent) => {
-        e.preventDefault()
-        console.info('submit placeholder')
-
+    const onSubmit = async (values: EQROSubmissionTypeFormValues) => {
         if (isNewSubmission) {
             const input: CreateContractInput = {
-                populationCovered: 'MEDICAID',
-                programIDs: [allPrograms[0].id],
+                // TODO: populate with real values / update types
+                populationCovered: values.populationCovered!,
+                programIDs: values.programIDs,
+                // managedCareEntities: values.managedCareEntities,
```

Comment:
Right now createDraft isn't setup to take in managedCareEntities. Is that work captured in another ticket or is that something that should be handled here?